### DISCUSSION TITLE
Project Without Repo

### DISCUSSION THREAD

**Discussion by macMikey** [community/community#42963] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

The thing I'm working on (which is technically a project, but, for the sake of not confusing everyone, I'm going to refer to it as "thing", and "project" as GH Projects) has multiple repos. Each of those repos have their own project.
I would like to be able to have a master project which I can use to track the macro tasks, which are then broken into issues and tasks in each of the repo projects.
Unfortunately, without tying the master project to a particular repo, i can't get any of the cards out of draft mode, which means i would have to create another repo, which is going to be completely empty of anything, just to make the master project work.
Then, of course, I'd also like to be able to take the subproject tasks/cards/issues, and post **some** of them to the master project, because, in many cases, those cards/tasks/issues will instead be check items in a list in a comment of a card in the master project.

**Comment by jfullstackdev** [comment]
that seems to be of how you design your projects, I'm wondering how large are your projects and why you ended up with that design,

in my current company, it's only one Org that is used to manage these repos and please do note this is an industry level, we do develop software for business, yet we are not even using GitHub Projects, we use mainly the Trello and sometimes, in a certain team, Google Spreadsheet, and we are efficient with that 

so you can organize your thoughts and innovate how you use the GitHub Project feature

**Comment by macMikey** [comment]
because they are different platforms, and the codebases are not similar. they are only related to each other by the macro goals.
